test/integration/scheduler/queue_test.go

```diff
@@ -197,10 +269,12 @@ func TestCoreResourceEnqueue(t *testing.T) {
 		// pods are the list of Pods to be created.
 		// All of them are expected to be unschedulable at first.
 		pods []*v1.Pod
+		// customePlugins are the custom pluins that will be enabled in the test case
+		customePlugins []string
 		// triggerFn is the function that triggers the event to move Pods.
 		triggerFn func(testCtx *testutils.TestContext) error
-		// wantRequeuedPods is the map of Pods that are expected to be requeued after triggerFn.
-		wantRequeuedPods sets.Set[string]
+		// wantRequeuedPodsByQHint is the map of Pods that are expected to be requeued after triggerFn stored by whether QHint is enabled
+		wantRequeuedPodsByQHint map[bool]sets.Set[string]
 		// enableSchedulingQueueHint indicates which feature gate value(s) the test case should run with.
 		// By default, it's {true, false}
 		enableSchedulingQueueHint []bool
```

Comment:
Let's merge these two fields, so make keys of `wantRequeuedPodsByQHint` enabling/disabling queueing hints feature flag.

Reply:
great idea!